test/src/unittests/rhythm/test_onsetdetectionglobal.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
In this file, we are missing a regression test on real audio.

Reply:
this will be added. The set of tests will be testZero, testRegression and testInvalidParam.